Umbraco-StarterKit/Creating-Content/index.md

```diff
@@ -3,3 +3,32 @@ versionFrom: 8.0.0
 ---
 
 # Creating content
+
+When you are creating content in Umbraco Uno, you will have many tools at your disposal, this page is designed to help you get started on utilizing these tools.
+
+## Getting started
+
+When you want to create content, it is a good idea to think about what tools to use for the job. An example of this is between image widget and gallery widget, image widget only allows you to add one image. However, if you only need one image that is the perfect tool for the job. Gallery widget will allow you to set up a gallery of images. That also means that you can choose to only select one image with that widget, which of these two widgets you choose to use for the job, depends on what you specifically need for this situation.
+
+The way to best achieve a good content creation flow could be to sit down mark up what you want to create on the page and then choose the tools accordingly.
+When you have chosen what should be on the page and what tools to use, then you can start building from the top to the bottom.
+
+### Manage Media Library
+
+When creating content, you will most likely need to work with media as well, and to best work, with your media library, this article offers a good piece of insight into how you can do this best[Manage Media Library](Manage-Media-Library/index.md).
+
+### Rich Text Editors
+
+The Rich Text Editor or RTE as people call it is a great way to create content, it is a text editor, that allows you to write longer text pieces with all the awesome formating you would need.
+
+To learn more about what features a  [Rich Text Editor](Rich-Text-Editors/index.md)  offers, it would be a good idea to read this article.
```

Comment:
```suggestion
To learn more about what features the RTE offers, it would be a good idea to read the [Rich Text Editor](Rich-Text-Editors/index.md) article.
```